pkg/systemd/quadlet/quadlet.go

```diff
@@ -1650,6 +1651,11 @@ func ConvertPod(podUnit *parser.UnitFile, name string, unitsInfoMap map[string]*
 	execStartPre.add("--infra-name", fmt.Sprintf("%s-infra", podName))
 	execStartPre.add("--name", podName)
 
+	shmSize, exists := podUnit.Lookup(PodGroup, KeyShmSize)
+	if exists {
```

Comment:
To align with other checks, please also check that there's a value:
```suggestion
	if exists && len(shmSize) > 0 {
```